### PR TITLE
Add checkout instructions to PR body

### PR DESCRIPTION
**Pull request by JornVernee**
Hi,

I'd like let the pr bot add the fetch comamnd found in the automatic emails to the PR body, so that people using only GitHub can see the instructions as well.

This adds a new "Instructions" section to the body of the PR message:

> ## Instructions
> To checkout these changes locally:
> ```bash
> $ git fetch https://git.openjdk.java.net/panama-foreign pull/16/head:pull/16
> $ git checkout pull/16
> ```

Testing:
- Running `.\gradlew test`

Any other recommendations for testing this?

Thanks,
Jorn

## Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

## Reviewers
 * Erik Helin ([ehelin](@edvbld) - **Reviewer**)

### CODE REVIEW

**Comment by edvbld** [comment]
Thanks Jorn, good idea :+1: I think we might want to overhaul the design a little bit to accommodate for this new section. I added in some more section that might appear in a pull request -  what do you think about the following?

> -----------------------
> ### Progress
> - [x] Change must not contain extraneous whitespace
> - [ ] Change must be properly reviewed (1/2)
> 
> ### Reviewers
> * Erik Duveblad ([ehelin](localhost) - **Reviewer**)
> * Joe Darcy ([darcy](locahost) - **Reviewer**) :arrows_counterclockwise: Re-review required (review applies to 7de15f5c44a6f4f5a18afda090d52b7b69c7d09c)
>
> ### Issue
> * [JDK-8712393](localhost): Add download instructions to pull request
>
>
> ### Contributors
> * Robin Westberg ([rwestberg](localhost))
>
> ### Download
> `$ git fetch https://git.openjdk.java.net/panama-foreign pull/16/head:pull/16`
> `$ git checkout pull/16`

**Comment by JornVernee** [comment]
Hey Erik,

I like the new format that you suggest as well :) I've applied the suggestions.

File: bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java [hunk 1]
```diff
@@ -302,7 +302,8 @@ private String formatContributor(EmailAddress contributor) {
 
     private String getStatusMessage(List<Comment> comments, List<Review> reviews, PullRequestCheckIssueVisitor visitor) {
         var progressBody = new StringBuilder();
-        progressBody.append("## Progress\n");
+        progressBody.append("</hr>");
```
Comment (edvbld) [inline bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java]:
```suggestion
        progressBody.append("---------");
```

**Comment by edvbld** [comment]
It seems like the test `org.openjdk.skara.bots.pr.ContributorTests > prBodyUpdates` is failing on both Linux and Mac. I think you have these two lines:

- https://github.com/openjdk/skara/blob/master/bots/pr/src/test/java/org/openjdk/skara/bots/pr/ContributorTests.java#L335
   `                if (line.equals("## Contributors")) {`
- https://github.com/openjdk/skara/blob/master/bots/pr/src/test/java/org/openjdk/skara/bots/pr/ContributorTests.java#L355
   `                assertNotEquals("## Contributors", line);`

**Comment by JornVernee** [comment]
@edvbld Thanks Erik,

That test is being skipped when running on Windows and I'm seeing a lot of "STANDARD_ERROR"s in the build log for the workflow, so not sure if this was a false negative.

I gave the latest version a run on WSL. There are a lot of failing tests if I do that, unrelated to this PR, but at least the particular test is now passing.

**Comment by edvbld** [comment]
@JornVernee Yeah, most of the bot tests are disabled on Windows. I would be happy to review patches to get the tests passing on WSL :+1:

**Pull request closed without merging**